tests/core/gui/test_validators.py

```diff
@@ -284,9 +285,11 @@ def test_file_name_validator(qtbot, core_config):
 )
 def test_user_name_validator(qtbot, core_config, name_and_expected):
     switch_language(core_config, "en")
-
     user_name, expected = name_and_expected
 
+    if IS_OXIDIZED and not expected:
+        pytest.xfail("TODO: Add validation for user name in rust")
+
```

Comment:
use [conditional parameter for `xfail`](https://docs.pytest.org/en/7.1.x/how-to/skipping.html#condition-parameter)

Reply:
e222d41bd71b49becac04b7170b75e4a8ac4a92b